jenkins/debian/debos/scripts/stretchltp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Please rename this file to `stretch-ltp.sh`, following the convention with other file systems.